code/datums/abilities/vampire.dm

```diff
@@ -239,6 +241,25 @@
 				owner.full_heal()
 			else
 				changeling_super_heal_step(healed = owner, mult = mult*2, changer = 0)
+		// alerts thralls when dead and not in coffin
+		else if (isdead(owner) && !death_triggered)
```

Comment:
instead of checking if they are dead every life tick it would be probably be better to make use of `COMSIG_MOB_DEATH`